topic-operator/src/main/java/io/strimzi/operator/topic/ReplicasChangeHandler.java

```diff
@@ -92,7 +100,9 @@ public List<ReconcilableTopic> requestPendingChanges(List<ReconcilableTopic> rec
         try {
             LOGGER.debugOp("Sending topic configuration request, topics {}", topicNames(reconcilableTopics));
             List<KafkaTopic> kafkaTopics = reconcilableTopics.stream().map(rt -> rt.kt()).collect(Collectors.toList());
+            Timer.Sample timerSample = TopicOperatorUtil.startExternalRequestTimer(metrics, config.enableAdditionalMetrics());
             String userTaskId = cruiseControlClient.topicConfiguration(kafkaTopics);
+            TopicOperatorUtil.stopExternalRequestTimer(timerSample, metrics::cruiseControlTopicConfig, config.enableAdditionalMetrics(), config.namespace());
```

Comment:
I'd guess you should also stop the time if an exception is thrown?

Reply:
Right. Thanks for catching this.